### DISCUSSION TITLE
Problemi con Alexa Media Player

### DISCUSSION THREAD

**Discussion by marcoferroha** [indomus/forum#1086] [HUB: Home Assistant]
Ho installato il repository per Hacs senza problemi.
Quando provo ad installare l'integrazione ricevo il messaggio:

Impossibile caricare il flusso di configurazione

Qualche idea per risolvere?

Marco Ferro

**Comment by snakuzzo** [comment]
Se hai installato stamattina...attento che l'ultima versione non è compatibile con l'attuale versione di HA

**Reply by marcoferroha** [reply]
no ha cominciato 3 versioni fa

**Reply by pirata880** [reply]
Quindi sulla versione 2021.11.5 Alexa Media Player non è più integrabile?
Io ho sistemato haaksa con aws ed è tutto ok. Ma scaricando "Alexa media Player" non lo trovo nelle integrazioni.
Il motivo è questo quindi!

**Reply by snakuzzo** [reply]
Scusa ma ora che c'entra haaska? Sono due cose completamente slegate.
haaska e alexa media player c'entrano come la nutella con le alici

**Reply by pirata880** [reply]
Era per precisare che ho integrato HA su Amazon Alexa via haaksa e grazie alla guida è andato tutto bene.
Poi dato che volevo utilizzare alexa anche come sistema di  notifica, e sulla guida si raccomanda di eseguire prima l'integrazione con "Alexa media player", procedendo in questo, mi sono accorto che quest'ultima non mi compare nelle integrazioni. Credevo, nella mia ignoranza, che fossero legate, sorry.

**Reply by pirata880** [reply]
Per dovere di aggiornamento, l'integrazione con Alexa Media Player è andata a buon fine. Naturalmente era un mio errore, non avevo compreso che per integrarla dovevo, dopo averla installata, cercarla anche nelle integrazioni, mentre io pensavo di trovarmela bella e pronta. Fatto questo poi ho solo avuto i soliti problemi con l'2FA. Ma dopo svariati tentativi tutto Ok.
Grazie

**Reply by snakuzzo** [reply]
...senza parole 😅
